registry/server/apps/routes/createApp.ts

```diff
@@ -22,6 +24,33 @@ const validateRequestBeforeCreateApp = validateRequestFactory([{
 const createApp = async (req: Request, res: Response): Promise<void> => {
     const app = req.body;
 
+    if (!app.spaBundle && app.assetsDiscoveryUrl) {
+        let response;
+
+        try {
+            response = await axios.get(app.assetsDiscoveryUrl, {responseType: 'json'});
+        } catch (error) {
+            console.log('Caught an error while trying to fetch a manifest file:');
```

Comment:
add link(`app.assetsDiscoveryUrl`) to the end of the `console.log`

Reply:
Done